x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -197,3 +227,70 @@ describe('Resolver, when analyzing a tree that has two related events for the or
     });
   });
 });
+
+/**
+ * Get the integer in a CSS px unit string
+ * @param px a string with `px` preceded by numbers
+ */
+function pxNum(px: string): number {
+  return parseInt(px.match(/\d+/)![0], 10);
+}
+
+/**
+ * Get computed boundaries for process node elements
+ */
+function getComputedNodeBoundaries(entityID: string) {
+  const { left, top, width, height } = getComputedStyle(
+    simulator.processNodeElements({ entityID }).first().getDOMNode()
```

Comment:
why is `.first()` needed here?

Reply:
`getDOMNode` will throw if it gets a collection instead of one thing. Since it's a test, we probably _want_ it to throw if that ever happens (since we wouldn't expect more than one processNodeElement for a given entityID. I'll pull that out - thanks for the spot.

Reply:
https://github.com/elastic/kibana/pull/75010/commits/3fe9e12545716f868fc1445c75fadb95428db1cd